sharding-proxy/src/test/java/io/shardingsphere/shardingproxy/backend/jdbc/connection/BackendTransactionManagerTest.java

```diff
@@ -39,10 +37,10 @@
     
     private BackendTransactionManager backendTransactionManager = new BackendTransactionManager(backendConnection);
     
-    @BeforeClass
-    public static void beforeClass() {
-        ShardingTransactionHandlerRegistry.load();
-    }
+//    @BeforeClass
+//    public static void beforeClass() {
+//        ShardingTransactionHandlerRegistry.load();
```

Comment:
Why comment here, should remove comment codes

Reply:
yes ,this can be removed.